docs/tracing.rst

```diff
@@ -249,24 +250,116 @@ runId              The workflow execution unique ID.
 event              The workflow execution event. One of ``started``, ``process_submitted``, ``process_started``, ``process_completed``, ``error``, ``completed``.
 utcTime            The UTC timestamp in ISO 8601 format.
 trace              A process runtime information as described in the :ref:`trace fields<trace-fields>` section. This attribute is only provided for the following events: ``process_submitted``, ``process_started``, ``process_completed``, ``error``.
+metadata           The workflow metadata including the :ref:`config manifest<config-manifest>`. This attribute is only provided for the following events: ``started``, ``completed``.
 ================== ================
```

Comment:
In the description on the metadata there's much more of the manifest now. Maybe make sense just to mention to the look the example below for the object structure.